cmd/mrpeek.cpp

```diff
@@ -288,6 +288,7 @@ void run ()
         case '+': image_scale *= 1.1; std::cout << VT::ClearScreen; break;
         case '-': image_scale /= 1.1; std::cout << VT::ClearScreen; break;
         case VT::MouseMoveLeft: focus[x_axis] += xp-x; focus[y_axis] += yp-y; break;
+        case VT::Escape: colourmap.update_scaling (x, y); break;
```

Comment:
This doesn't seem to work... But more to the point, the Home key already does this. See commit 5271a76.

Reply:
Actually, the Esc key seems to be handled reliably all of a sudden... I'll fix it to use the Esc key and remove the Home key binding.

Reply:
I added it as Home is Fn + Left Arrow on a macbook and the right click mouse drag was quite slow still